### ISSUE TITLE
Add PI's 9B and 2S to CSBCD router port forwarding

### ISSUE THREAD

**Issue by carlosparadis** [erdl/sensors#20]
For the record, this was wrapped out today around 7 PM. 

## Missing steps for the record

To add port forwarding, refer to the internal Google Sites of the group, and look for the "Router" section at the left menu. There is a chance you may not have access to it. If you need port forwarding, let me know and I can revisit this.

 - In the page, there will be in the screenshot the IP to access the router. Eileen has the root password access to it, which is required to edit port forwarding.  
 - Your computer mac address also needs to be added to the router by **another computer with mac address** for you even to be able to login to CSBCD network (required for the router IP to work on your browser!). To add more computers to the list of accepted mac address, this is also the place to go. 

As far as not having wi-fi in the PI, I have added to the wiki how to fix that by adding all users to the pi group ([as shown in the wiki revision](https://github.com/erdl/sensors/wiki/Raspbian-Setup,-Security-and-Remote-Access/_history)). 

## TL;DR

 * You should be able to SSH on both units in the lab from anywhere you are, including your home. **Make sure you change your passwords on both units** ASAP. 
 * @jygh98 was given sudo to unit 2B (old pi) by me, but only I have sudo to the new unit 9S. This is intended to monitor authentication traffic. Jason also agreed to:
   *  not do anything as root user, so as not to mix the history log
   * not do anything unrelated to sensor code (such as setting a static ip).

**Comment by ryantanaka** [comment]
@carlosparadis, i changed my pw on pi 2B so thats all set
for pi 9S, i don't think my public key is on there. can you throw it on when you get the chance? 
```
Permission denied (publickey).
```
thanks
